libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1039,7 +1044,7 @@ bool AP_InertialSensor::calibrate_trim(float &trim_roll, float &trim_pitch)
 bool AP_InertialSensor::accel_calibrated_ok_all() const
 {
     // calibration is not applicable for HIL mode
-    if (_hil_mode) {
+    if (_hil_mode || _sitl_mode) {
```

Comment:
this would break using SITL for testing accelcal

Reply:
Got it... Well, AP_InertialSensor is unaware of EKF type, so the only other way of having AP_InertialSensor report OK health with 0 offsets in SITL is to pass EKF type to accel_calibrated_ok_all(). Are you OK with that solution?

Reply:
it could have an #ifdef for SITL and check EKF type. Similar to what I did for this PR:
https://github.com/ArduPilot/ardupilot/pull/11634

Reply:
OK, looks like that PR fixed this, I'll revert my change to AP_InertialSensor.cpp